Jellyfin.Api/Controllers/DynamicHlsController.cs

```diff
@@ -41,6 +42,9 @@ namespace Jellyfin.Api.Controllers
     [Authorize(Policy = Policies.DefaultAuthorization)]
     public class DynamicHlsController : BaseJellyfinApiController
     {
+        private const string DefaultEncoderPreset = "veryfast";
+        private const TranscodingJobType _transcodingJobType = TranscodingJobType.Hls;
```

Comment:
Very minor thing, but can you rename it back to `TranscodingJobType`?

Reply:
Done.